build-system/common/check-package-manager.js

```diff
@@ -331,18 +339,21 @@ function checkPythonVersion() {
         cyan(recommendedVersion) + yellow('.')
       );
       console.log(
-        yellow('⤷ To fix this, install the correct version from'),
-        cyan(`https://www.python.org/download/releases/${recommendedVersion}`) +
-          yellow('.')
+        yellow('⤷ To fix this, install a supported version from'),
+        cyan('https://www.python.org/downloads') + yellow('.')
       );
     }
   } else {
     console.log(
-      yellow(
-        'WARNING: ' +
-          'Could not determine the local version of python. ' +
-          'AMP development requires python 2.7.'
-      )
+      yellow('WARNING: Could not determine the local version of python.')
+    );
+    console.log(
+      yellow('⤷ To fix this, make sure'),
+      cyan(pythonExecutable),
+      yellow('is in your'),
+      cyan('PATH'),
+      yello('and is version'),
```

Comment:
`yello` -> `yellow`

Reply:
I knew I should have waited 'til lunch to use my laptop...